manifests/w/WindRiver/StudioCLI/23.3.4/WindRiver.StudioCLI.installer.yaml

```diff
@@ -17,6 +17,8 @@ Installers:
     PortableCommandAlias: mc
   - RelativeFilePath: vault.exe
     PortableCommandAlias: vault
+  - RelativeFilePath: gitlab.exe
```

Comment:
```suggestion
  - RelativeFilePath: glab.exe
```

Reply:
Thank you for your suggestion. It is the problem caused by the storage of our package not updated. I will close this PR